value-fixture/test/org/immutables/fixture/generatorext/RewriteTest.java

```diff
@@ -15,13 +15,15 @@
  */
 package org.immutables.fixture.generatorext;
 
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class RewriteTest {
   @SuppressWarnings("CheckReturnValue")
   @Test
+  @Disabled("FIXME: This test doesn't pass on either of provided JDKs")
```

Comment:
This test says that it should throw IllegalArgumentException but it always throws NullPointerException. On all JDKs. I don't know who to make it pass.

Reply:
the idea here is that generated code have imports rewritten to point to another preconditions 
```java
@Metainf.Service
public class PreconditionsRewriter implements GeneratedImportsModifier {
```

```java
  static <T> T requireNonNull(T instance) {
    if (instance == null) {
      throw new IllegalArgumentException();
    }
    return instance;
  }
```

Reply:
seems like `META-INF/services` are not working (any more) is some contexts. (this issue, also custom annotations etc)
Can somebody help a little bit, maybe there are some known information wrt maven/java/annotation processing changes in this area?

Reply:
the same here, we can just change fixme to point to `GeneratedImportsModifier` instances are not loaded

Reply:
or fix the real issue if we know it

Reply:
probably affected by annotationProcessorPath change